### DISCUSSION TITLE
Zigbee CC26x2R1 - WARN NPM fsevents - Have to do something? I can't find more informations

### DISCUSSION THREAD

**Discussion by M1cS0w** [Koenkk/zigbee2mqtt#5437] [General]
Hello,

I got warnings while deinstall adapters in iobroker:
![Screen_NPM](https://user-images.githubusercontent.com/51213428/103242364-37603880-4956-11eb-99a1-7797a368eaad.jpg)

I have used the deconz adapter with conbee II, but now i have changed to zigbee with the new ti board. So i don't know, if it is normal or what I have to do.

**Comment by Koenkk** [comment]
Completely normal and expected. no need to worry

**Reply by M1cS0w** [reply]
Nice, thanks a lot Koenkk. Wow, so fast ;) Have a nice day. And perfect what u made.
I was confused, because this come while deinstall of any other adapters in iobroker, would it come in for example while install/deinstall zigbee, I would think in other way :P
